applications/nrf_desktop/src/modules/fast_pair.c

```diff
@@ -119,6 +143,17 @@ static bool handle_module_state_event(const struct module_state_event *event)
 	return false;
 }
 
+static bool handle_ble_dongle_peer_event(const struct ble_dongle_peer_event *event)
+{
+	dongle_app_id = event->bt_app_id;
+
+	if (selected_stack_id == dongle_app_id) {
```

Comment:
`if dongle_app_id == 0`
Please do not mix application and Bluetooth local identities (you can define default application local identity as zero to make this comparison clear).